sgkit/stats/association.py

```diff
@@ -114,20 +167,30 @@ def gwas_linear_regression(
     when not already present, but there is currently no parameterization for
     intercept-free regression.
 
+    Warning: Both covariate and trait arrays will be rechunked to have blocks
```

Comment:
You can make this a `Warning` heading so it's recognised as a numpydoc warning.

Reply:
Ok, I moved it in https://github.com/pystatgen/sgkit/pull/52/commits/32034c384049f9553a833619c92de76a2f174361.  Maybe there's a way to make the two separate warnings break into different warning boxes in the docs?  I'm having a hard time finding good examples.